microservices/client/src/components/buttons/PillButtons.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { IconType } from 'react-icons';
+
+
+
+interface PillButtonProps {
```

Comment:
convertion for definiting interfaces is always to start with an `I` in front
```suggestion
interface IPillButtonProps {
```